Directory.Packages.props

```diff
@@ -28,16 +23,18 @@
         3) Since version 3.1.0, the .NET runtime team is holding a high bar for backward compatibility on
           these packages even during major version bumps, so compatibility is not a concern here.
     -->
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="$(LatestRuntimeOutOfBandVer)" />
```

Comment:
How do we plan to address one or two of these packages making higher versions available than the rest?

Reply:
If you need to use an explicit version for something, you replace `$(LatestRuntimeOutOfBandVer)` with the version you need to use. There's no rule that says everything must be on the same version 😄

Let's say there is a patch of `Microsoft.Extensions.Logging.Configuration` we need to take: 

```diff
    <PackageVersion Include="Microsoft.Extensions.Logging" Version="$(LatestRuntimeOutOfBandVer)" />
-   <PackageVersion Include="Microsoft.Extensions.Logging.Configuration" Version="$(LatestRuntimeOutOfBandVer)" />
+   <PackageVersion Include="Microsoft.Extensions.Logging.Configuration" Version="9.0.1" />
```

Could rename the prop to something else if you think it would be more clear?

Reply:
Just leave a comment that this could happen, and later, when a common version is available, it needs to switch to `$(LatestRuntimeOutOfBandVer)`. This PR (https://github.com/open-telemetry/opentelemetry-dotnet/pull/5853/files) shows that a few versions were different, but they all eventually converged to a common version with .NET 9.0.

Reply:
I added a note in there. LMK what you think.